README.md

```diff
@@ -38,20 +52,70 @@ for chunk in stream:
   print(chunk['message']['content'], end='', flush=True)
 ```
 
+## Custom client
+A custom client can be created with the fields from the [httpx client](https://www.python-httpx.org/api/#client).
```

Comment:
This reads weird. Focus on a custom Ollama client until `extra arguments are passed to httpx.Client` at the end

Reply:
Yeah reading this again it doesn't read well- will workshop